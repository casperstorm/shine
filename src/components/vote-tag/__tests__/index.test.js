import renderer from 'react-test-renderer'
import React from 'react'

import Vote from '../index'

describe('vote', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Vote title={'foo'} type={'positive'} />)
    expect(tree).toMatchSnapshot()
  })
})
