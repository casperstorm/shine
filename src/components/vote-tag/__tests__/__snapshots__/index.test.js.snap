// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`vote should render correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "borderRadius": 10,
        "flexDirection": "row",
        "height": 23,
        "justifyContent": "center",
        "marginRight": 7,
      },
      undefined,
    ]
  }
>
  <View
    style={
      Object {
        "marginLeft": 8,
        "marginTop": 1,
      }
    }
  >
    <Image
      source={1}
    />
  </View>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#ffffff",
        "fontFamily": "Rubik-Regular",
        "fontSize": 12,
        "marginLeft": 8,
        "marginRight": 8,
      }
    }
  >
    foo
  </Text>
</View>
`;
